reputation/reputation.go

```diff
@@ -0,0 +1,180 @@
+package reputation
```

Comment:
It would be great to have some tests on the reputation package. To assure the calculation makes sense and if wrong values are provided, they are correctly managed.